src/bundle/construct-manifest.ts

```diff
@@ -44,7 +45,8 @@ const routeMapWithHandlers = {
 
 const edgeSpec = {
   routeMatcher: getRouteMatcher(Object.keys(routeMapWithHandlers)),
-  routeMapWithHandlers
+  routeMapWithHandlers,
+  makeRequest: async (req, options) => handleRequestWithEdgeSpec(edgeSpec, options)(req)
```

Comment:
Could this wrap around the existing module service code? The code there is slightly different than a normal request, and for instance handles route parameter identification automatically

Reply:
👍 will try to add base path identification (I think that's what you mean?)

I figured that all adapters should be sharing the same logic here (want to allow Node/Bun adapters to remove the base path too) and imo a single function call is slightly easier than having to construct a new instance

Reply:
> (I think that's what you mean?)

yeah sorry for unclear

> I figured that all adapters should be sharing the same logic here (want to allow Node/Bun adapters to remove the base path too) and imo a single function call is slightly easier than having to construct a new instance

yeah that's reasonable, I just wanna make sure that that code i wrote doesn't go to waste

you should probably also remove the old module service code too while you're at it, rather than just the tests